packages/eas-cli/src/commands/update/index.ts

```diff
@@ -586,15 +613,43 @@ async function getRuntimeVersionObjectAsync(
     }
   }
 
-  return Object.fromEntries(
-    platforms.map(platform => [
-      platform,
-      nullthrows(
-        Updates.getRuntimeVersion(exp, platform),
-        `Unable to determine runtime version for ${platform}`
-      ),
-    ])
-  );
+  try {
+    return [transformRuntimeVersions(exp, platforms), null];
+  } catch (error: any) {
+    if (nonInteractive) {
+      Errors.error(error, { exit: 1 });
+    }
+
+    ora().start().fail(error.message);
```

Comment:
What's that? Do you want to print the `x` sign? You can do it explicitly. See how we implemented `Log.succeed` - https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/log.ts#L48

Reply:
There was not a helper for that, so I have used Ora directly.

I always attempt to reduce the amount of lines changed, if not needed, but probably we can add that helper, since this PR evolved fince it's first form already.

Reply:
Let's change that